core/src/main/java/org/mule/runtime/core/el/DataWeaveExpressionLanguage.java

```diff
@@ -45,14 +51,24 @@
   public static final String ERROR = "error";
   public static final String VARIABLES = "variables";
   public static final String FLOW = "flow";
+  public static final String SERVER = "server";
+  public static final String MULE = "mule";
+  public static final String APP = "app";
 
   private ExpressionExecutor expressionExecutor;
+  private MuleContext muleContext;
 
   @Inject
-  public DataWeaveExpressionLanguage(ExpressionExecutor expressionExecutor) {
-    this.expressionExecutor = expressionExecutor;
+  public DataWeaveExpressionLanguage(MuleContext muleContext) {
+    try {
+      this.expressionExecutor = muleContext.getRegistry().lookupObject(ExpressionExecutor.class);
+    } catch (RegistrationException e) {
+      throw new MuleRuntimeException(e);
```

Comment:
Add a descriptive message saying the executor could not be obtained or something